src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1129,7 +1150,8 @@ private static byte[] getData(Profile p, int tagSignature) {
      * The array should contain data in a format, corresponded to the
      * {@code tagSignature} as defined in the ICC specification, section 10.
      * This method is useful for advanced applications which need to access
-     * profile data directly.
+     * profile data directly. Only non-built-in, application provided profiles
+     * should be updated using this method.
```

Comment:
What you mean is that only non-built-in profiles *can* be updated. The fix makes it impossible to update built-in profiles.

Reply:
I'll be inverting this statement to state what is NOT allowed rather than what is allowed by .setData() as below. Does the following sound better?

```
     * Note: JDK built-in ICC Profiles cannot be updated using this method
     * as it will result in IAE. JDK built-in profiles are those obtained by
     * {@code ICC_Profile.getInstance(int colorSpaceID)} where colorSpaceID
     * is one of the following:
     * {@link ColorSpace#CS_sRGB}, {@link ColorSpace#CS_LINEAR_RGB},
     * {@link ColorSpace#CS_PYCC}, {@link ColorSpace#CS_GRAY} or
     * {@link ColorSpace#CS_CIEXYZ}
```

Reply:
This is clearer.

Reply:
Looks like my previous commit was not reflected. I have update it now.

@prrace Updated PR and CSR.